package.json

```diff
@@ -12,8 +12,10 @@
     "build:dev": "wireit",
     "build:prod": "wireit",
     "start:docker": "docker-compose -f docker/docker-compose.yml up --build --remove-orphans",
-    "watch:dev": "npm run build:dev && (npm run start:dev > /dev/null 2>&1 & WIREIT_FAILURES=continue npm run build-and-check:dev --watch)",
-    "watch:prod": "npm run build:prod && (npm run start:prod & WIREIT_FAILURES=continue npm run build-and-check:prod --watch)",
+    "watch:dev": "npm run start-and-check:dev --watch",
```

Comment:
Do you think we need this pattern of a specific watch command or should we just use `--watch` on the command line?

Reply:
That seems reasonable.

Actually how about this? `watch:dev` is now just `start`, and `watch:prod` is `start:prod`.

So to spin it up locally, you just write `npm start --watch`